benches/benches/vm_initialization.rs

```diff
@@ -88,7 +88,7 @@ pub fn vm_initialization(c: &mut Criterion) {
     loop {
         let size = 8 * (1 << i);
         if size as u64 > consensus_params.script_params.max_script_data_length {
-            break
+            break;
```

Comment:
Is it possible to remove changes like that from the PR to minimise the number of conflicts in the future?

Reply:
Seems like my formatter did something there. I got it removed, thanks!